src/Pool.sol

```diff
@@ -56,6 +61,11 @@ abstract contract Pool {
     /// @notice Maximum number of receivers of a single sender.
     /// Limits costs of changes in sender's configuration.
     uint32 public constant MAX_RECEIVERS = 100;
+    /// @notice Maximum number of drip sreceivers of a single user.
+    /// Limits costs of dripping.
+    uint32 public constant MAX_DRIPS_RECEIVERS = 200;
```

Comment:
:1

Reply:
I hope it's a `:+1:`, not `:-1:` :laughing:

Reply:
haha